discord/enums.py

```diff
@@ -1020,6 +1020,12 @@ class EntitlementOwnerType(Enum):
     user = 2
 
 
+class PollLayoutType(Enum):
+    """The poll's layout type. Only one exists for now"""
```

Comment:
doesn't need to be stated, it'll just get removed once they add more (if they do lol)
```suggestion
    """The poll's layout type."""
```